### PR TITLE
Add capture similar element doc

### PR DESCRIPTION
**Pull request by jasonxuclickcorp**

### CODE REVIEW

File: doc/developtools/vscode/recorder/capture_similar_element.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Recapture
+Recapture similar element can be used to update your single locator or similar locator.
+- Start from Visual Studio Code   
+![vscode recapture](../../../img/recorder_recapture_vscode.png)
+- Recapture winow   
+![recapture window from VSC](../../../img/recapture_window.png)
+- Capture similar element icon
+![capture similar element icon](../../../img/recapture_similar_element_icon.png)
+- [Start capture](#start-capture)
```
Comment (robinbob) [inline doc/developtools/vscode/recorder/capture_similar_element.md:82]: "start capture" ?

Comment (jasonxuclickcorp) [reply doc/developtools/vscode/recorder/capture_similar_element.md]: update to capture similar elements steps.

File: doc/developtools/vscode/recorder/recorder.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
-# Clicknium Recorder 
+# Clicknium Recorder
 
 Clicknium Recorder provides operations for recording locators, recapturing locators, validating locators and locator management.
 
 - [Quick Start](./quickstart.md)
-- [Capture](./capture.md)
-- [Recapture](./recapture.md)
+- [Capture Single Element](./capture.md)
+- [Capture Similar Element](./capture_similar_element.md)
```
Comment (robinbob) [inline doc/developtools/vscode/recorder/recorder.md:7]: Capture Similar Elements

File: doc/developtools/vscode/recorder/capture_similar_element.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+## Start recorder
+- Start from Visual Studio Code LOCATORS tab  
+![start from vscode local locator](../../../img/start_recorder_from_vscode.png)
+- Strat from Visual Studio Code CLOUD LOCATORS tab  
+![start from vscode cloud locator](../../../img/start_recorder_from_cloud.png)
+- Start with keyboard shortcut `Ctrl+F10` in code.   
+
+## Capture wizard  
+Click on the `Similar elements` button, a wizard window will be displayed.
+- Clikc  `Similar elements` button
```
Comment (robinbob) [inline doc/developtools/vscode/recorder/capture_similar_element.md:33]: Click

File: doc/developtools/vscode/recorder/capture_similar_element.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+- Capture similar elements wizard 
+
+![capture similar element wizard](../../../img/capture_similar_element_wizard.png)
+
+
+## Start capture
+
+1. Capture first element.   
+Move mouse cursor to one UI element of target list, it will highlight the element, then press `Ctrl+click` to capture the locator. The element image will be added to the wizard. A locator for this element will be generated accordingly and matched elements count for this locator will be displayed below the image.  
+![capture first element](../../../img/capture_first_similar_element.png)
```
Comment (robinbob) [inline doc/developtools/vscode/recorder/capture_similar_element.md:45]: Better use another image. the screenshot is same as the document, readers may get confused when reading the document. You can use sample like searching engine results, or list of items in amazon.

File: doc/developtools/vscode/recorder/capture_similar_element.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+- Click the `Cancel` button  
+![cancel capture similar element](../../../img/cancel_capture_similar_element.png)  
+
+## Recapture
+Recapture similar element can be used to update your single locator or similar locator.
+1. Start from Visual Studio Code   
+![vscode recapture](../../../img/recorder_recapture_vscode.png)  
+2. Click `Similar elements` button  
+![recapture similar element button](../../../img/recapture_similar_element_button.png)  
+3. Capture first element.
```
Comment (robinbob) [inline doc/developtools/vscode/recorder/capture_similar_element.md:82]: No need to repeat same content. 
3. Follow steps in [Start capture](..link...) to generate the locator. After clicking ·Save· button, the locator in store will be replaced by the recaptured one.

File: doc/developtools/vscode/recorder/recorder.md (hunk 2 above)
Comment (jasonxuclickcorp) [reply doc/developtools/vscode/recorder/recorder.md:7]: Fixed, thanks.

File: doc/developtools/vscode/recorder/capture_similar_element.md (hunk 3 above)
Comment (jasonxuclickcorp) [reply doc/developtools/vscode/recorder/capture_similar_element.md:33]: Fixed, thanks.

File: doc/developtools/vscode/recorder/capture_similar_element.md (hunk 5 above)
Comment (jasonxuclickcorp) [reply doc/developtools/vscode/recorder/capture_similar_element.md:82]: Updated, thanks.

File: doc/developtools/vscode/recorder/capture_similar_element.md (hunk 4 above)
Comment (jasonxuclickcorp) [reply doc/developtools/vscode/recorder/capture_similar_element.md]: Fixed, thanks.

File: clicknium.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+startUp: app
```
Comment (DiceyClickcorp) [inline clicknium.yaml:1]: del?

File: doc/developtools/vscode/recorder/capture_similar_elements.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+![highlight similar elements2](../../../img/highlight_similar_elements2.png)
+
+## Clear elements
+Clear all the elements captured in the wizard window by clicking the right-top `Clear` button. It is normally used if the matched elements are not expected and you want to startover. Once clicking the button, the elements are removed and the `Clear` and `Save` buttons are disabled when there is no data.
+- Click the `Clear` button
+![clear similar elements](../../../img/clear_similar_elements.png)
+
+## Remove element
+Remove one element in the wizard window.  
+- Click the `Remove` icon
```
Comment (DiceyClickcorp) [inline doc/developtools/vscode/recorder/capture_similar_elements.md:68]: Click the `Remove` button

File: doc/developtools/vscode/recorder/capture_similar_elements.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Highlight elements
+Highlight all matched elements for several seconds to determine if the generated locator is correct.  
+- Click the matched elements count button  
+![highlight similar elements1](../../../img/highlight_similar_elements1.png)
+![highlight similar elements2](../../../img/highlight_similar_elements2.png)
+
+## Clear elements
+Clear all the elements captured in the wizard window by clicking the right-top `Clear` button. It is normally used if the matched elements are not expected and you want to startover. Once clicking the button, the elements are removed and the `Clear` and `Save` buttons are disabled when there is no data.
+- Click the `Clear` button
```
Comment (DiceyClickcorp) [inline doc/developtools/vscode/recorder/capture_similar_elements.md:63]: new line with two space

File: doc/developtools/vscode/recorder/capture_similar_elements.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+![similar elements button](../../../img/similar_elements_button.png)
+- Capture similar elements wizard 
+
+![capture similar element wizard](../../../img/capture_similar_element_wizard.png)
+
+
+## Start capture
+
+1. Capture first element.
```
Comment (DiceyClickcorp) [inline doc/developtools/vscode/recorder/capture_similar_elements.md:43]: title not end with . 
-> Capture first element

**Pull request merged**